src/plugins/support.ts

```diff
@@ -18,8 +18,8 @@ export default fp<SupportPluginOptions>(async (fastify, opts) => {
     return roundPrice(getDiscount(price, discountPercentage))
   })
 
-  fastify.decorate('getBase64FromUrl', async function (url: string) : Promise<string>{
-    let imageUrlData = await fetch(url)
+  fastify.decorate('getBase64FromUrl', async function getBase64FromUrl(url: string) : Promise<string>{
```

Comment:
Stessa cosa qui', se vuoi usarle named, dichiarale in un altro file e importale altrimenti tienile anonymous